manifest.versions.json

```
@@ -5,15 +5,15 @@
     "base-url|public|preview|main": "$(base-url|public|maintenance|main)",
     "base-url|public|preview|nightly": "https://dotnetbuilds.azureedge.net/public",
 
-    "aspire-dashboard|8.2|build-version": "8.2.1-preview.1.24473.4",
-    "aspire-dashboard|8.2|product-version": "8.2.1",
-    "aspire-dashboard|8.2|fixed-tag": "$(aspire-dashboard|8.2|product-version)",
-    "aspire-dashboard|8.2|minor-tag": "8.2",
-    "aspire-dashboard|8|major-tag": "8",
-    "aspire-dashboard|8.2|linux|x64|sha": "df4c189f4fecb3284de436c04ab42aaf4888a0929f04c95ba3e1774e07045032cc8e77f570f2cf186fd2db12e385e866732dd98882f06ee0e5489d566d09b667",
-    "aspire-dashboard|8.2|linux|arm64|sha": "c6d0a9da8e459dd64f24758b60cc8b587cbf1c1300188f924d28aea3ace483858b1bb1287782121067af00b9d5533a529a070e8b376f85f7775ffd03c4b193fd",
-    "aspire-dashboard|8.2|base-url|main": "$(base-url|public|preview|nightly)",
-    "aspire-dashboard|8.2|base-url|nightly": "$(base-url|public|preview|nightly)",
+    "aspire-dashboard|9.0|build-version": "9.0.0-preview.5.24551.3",
+    "aspire-dashboard|9.0|product-version": "9.0.0-preview.5",
```

I was expecting this to contain the versions:

```suggestion
    "aspire-dashboard|9.0|build-version": "9.0.0",
    "aspire-dashboard|9.0|product-version": "9.0.0",
```

And for the shas to be for the binaries that will be released when it that version goes live (I did not check if these shas match version `9.0.0-preview.5`).

---

Updated based on your suggestion, and the shas should be accurate.